docker-vms/freeswitch/Dockerfile

```diff
@@ -0,0 +1,25 @@
+# Taken from https://freeswitch.org/confluence/display/FREESWITCH/Debian#Debian-DebianPackage
+
+FROM debian:wheezy
+MAINTAINER Praekelt Foundation <dev@praekeltfoundation.org>
+
+# Add the repository
+RUN echo 'deb http://files.freeswitch.org/repo/deb/debian/ wheezy main' \
+    >> /etc/apt/sources.list.d/freeswitch.list
+RUN gpg --keyserver pool.sks-keyservers.net --recv-key D76EDC7725E010CF
+RUN gpg -a --export D76EDC7725E010CF | apt-key add -
+
+# Install the packages
+ADD retry_until_pass.sh ./
+RUN ./retry_until_pass.sh apt-get update
```

Comment:
There are some quite big binaries (I think 3 that are >50mb), and on my local machine they kept on failing. But with the docker build, if you restart the build it has to start again from the beginning, redownloading all of these big files. If you rerun the `apt-get update` command like above, it doesn't need to redownload all that it has already.

I'm not sure if this will fail on servers, but it was very frustrating on my local machine to have to wait for it to redownload the same large files over and over, because one file failed to download (and it doesn't seem like apt-get does download retries)